kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -0,0 +1,96 @@
+server {
+    listen 80;
+
+    gzip                     on;
+    gzip_min_length          1000;
+    gzip_proxied             expired no-cache no-store private auth;
+    gzip_types               text/plain text/css text/javascript application/xml;
+    client_max_body_size     {{ max_upload_size }};
+
+
+    location /download/ {
+        alias /srv/storage/;
+    }
+
+    location = /upload/ {
+
+        access_by_lua_block {
+            -- Module nginx big upload support only content-type: "application/x-www-form-urlencoded"
```

Comment:
Does this module really suppport `application/x-www-form-urlencoded`? Or does it simply ignore the content type?

Reply:
I suppose that, when `$backend_url` variable was set in nginx config, request `content-type` header send by user change location to request body and request `content-type` header is set to `application/x-www-form-urlencoded`. You can see this https://github.com/pgaertig/nginx-big-upload/blob/master/lib/backend_file_storage_handler.lua

Reply:
This is not it.

If you look closely at the code you can see that `end_backend()` is called from within your `/upload-internal/` section, just when nginx finishes receiving the file from the client and before it starts sending it to the backend. The code you're pointing at sets the `Content-Type` header that **goes to the backend** - i.e. the `/result` section. The original `Content-Type` header gets overwritten. 

Before that (i.e. in `/upload/`) `nginx-big-upload` does not even look at this header.

So it's like I said. `/upload/ ignores `Content-Type`. Please update your description because it's misleading.

Reply:
Done.